### DISCUSSION TITLE
Blackberry 9520 (Blackberry OS 5)

### DISCUSSION THREAD

**Discussion by Computershik73** [mahomaps/mm-v1#28] [Tested devices]
Висит на заставке. Сеть есть через сим-карту (3G), Wi-Fi на РСТ версии нет.
![изображение](https://github.com/mahomaps/mm-v1/assets/31623340/0d6b3dc5-4397-4f72-b693-9b772b13c702)

**Comment by Feodor0090** [comment]
#39 решает проблему с зависанием (падало молча, теперь будет орать). Ждёт повторного теста.

**Reply by Computershik73** [reply]
Теперь не падает, пишет, что нет доступа к файловой системе к "file://C:/MahoMaps". В контекстном меню смог выбрать RMS, продвинулось чуть дальше до ошибки сети. Wi-Fi на этой модели для РСТ версии недоступен (неизвестно, можно ли включить программно). Предлагаю добавить диалог с выбором сети (вифи либо сотовой) при первом старте приложения и опцию в настройках для дальнейшего изменения при необходимости, issue создал.

**Reply by Feodor0090** [reply]
@Computershik73 если заинтересован - тащи посфикс коннекшн-строчки для симки. У тебя точно есть.

В противном случае так и оставим, т.к. ты единственный юзер ББ и это менее приоритетно чем маршруты/закладки/пробки/anything else

**Comment by Computershik73** [comment]
Теперь работает всё, кроме определения местоположения.
![изображение](https://github.com/mahomaps/mm-v1/assets/31623340/302b2387-14ae-458f-93d9-5b0b401262a5)
